Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -518,12 +520,12 @@ private async Task PerformPackageInstallation(IProgress<double> progress, string
                 return;
             }
 
-            if (target == null)
-            {
-                target = Path.Combine(_appPaths.PluginsPath, Path.GetFileNameWithoutExtension(package.targetFilename));
-            }
+            // Always override the passed-in target (which is a file) and figure it out again
```

Comment:
We should change the function signature then...

Reply:
That's far out of my element, I went for quick-and-dirty right now.

Reply:
Could you add a TODO then?

Reply:
Done!